MatrixSDKTests/MatrixSDKTestsData.h

```diff
@@ -45,7 +45,8 @@ FOUNDATION_EXPORT NSString * const kMXTestsAliceAvatarURL;
 
 // Get credentials asynchronously
 // The user will be created if needed
-- (void)getBobCredentials:(void (^)(void))success;
+- (void)getBobCredentials:(XCTestCase*)testCase
```

Comment:
We could use:
`- (void)getBobCredentialsFromTestCase:(XCTestCase*)testCase completion:(void (^)(void))completion;`